client/routes.jsx

```diff
@@ -24,6 +25,12 @@ const onRouteChange = (store) => {
   store.dispatch(stopSketch());
 };
 
+// TODO: Investigate using react-router for this switch
+// const ignoreMobile = () => window.location.search.substring(1).includes('ignoremobile');
+// const isMobile = () => window.innerWidth <= 760;
+// const IDEView = isMobile() && !ignoreMobile() ? IDEViewMobileScreen : IDEViewScreen;
+
+// How to use URL as a prop?
```

Comment:
@ghalestrilo What is it you want to use the URL for? Maybe I can help?

Reply:
@andrewn the idea here is to have a url parameter to render the desktop view even if the user is on a small screen.

Reply:
also @ghalestrilo if possible, i'd prefer to not merge in commented out code, so would it be possible to remove this for now?

Reply:
Hi guys, sorry for the delay.
Absolutely, will update this right now. @andrewn I had coded a way to bypass the `/mobile` view through a URL parameter, using `window.location`, and @catarak suggested we instead used the query parameter as a prop.
I wrote the comment because I didn't know how to do it at the moment, but I think following the react-router documentation will be simple enough :)

Reply:
I see. Yeah, there's no way to specify it in the `Route` and react-router doesn't automatically parse it ASFAIK.

I usually:
1. Get the [`location`](https://reacttraining.com/react-router/web/api/location) object passed into the mounted component (or the `withRouter` HoC, or the `useLocation` hook).
2. Pass `location.search` into the browser's native [`URLSearchParams` helper](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams)

What I've used in the past, is fetching the querystring from `location.search`